index.html

```diff
@@ -33,5 +40,124 @@ <h2>LET'S CONNECT</h2>
+            <li class="js">JavaScript</li>
+          </ul>
+          <button class="seepro">See project</button>
+        </div>
+        <div class="grid_item">
+          <img src="/Asset/oval.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul class="can">
+            <li class="canopy">CANOPY</li>
+            <li class="backend">Back End Dev</li>
+            <li class="twenty">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="lang">
+            <li class="html">HTML</li>
+            <li class="css">CSS</li>
+            <li class="js">JavaScript</li>
+          </ul>
+          <button class="seepro">See project</button>
+        </div>
+        <div class="grid_item">
+          <img src="/Asset/prof.svg" alt="" />
+          <h2>Multi-Post Stories</h2>
+          <ul class="can">
+            <li class="canopy">CANOPY</li>
+            <li class="backend">Back End Dev</li>
+            <li class="twenty">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="lang">
+            <li class="html">HTML</li>
+            <li class="css">CSS</li>
+            <li class="js">JavaScript</li>
+          </ul>
+          <button class="seepro">See project</button>
+        </div>
+      </div>
+    </section>
+    <section class="about">
+      <div class="about_me">
+        <h2>About MySelf</h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website. Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <h3>LET'S CONNECT</h3>
+      </div>
+      <div class="socialm">
+        <img src="/Asset/github.svg" alt="" />
+        <img src="/Asset/linkedin.svg" alt="" />
+        <img src="/Asset/angellist.svg" alt="" />
+        <img src="/Asset/twitter.svg" alt="" />
+        <img src="/Asset/meduim.svg" alt="" />
+      </div>
+      <div class="language">
+        <button class="gmr">Get my resume</button>
+        <h1>Languages</h1>
+        <img src="/Asset/arrow.svg" class="arrow" alt="" />
+        <ul class="lang_drop">
+          <li class="javas">
+            <img src="/Asset/js.svg" alt="JavaScript" /><span>JavaScript</span>
+          </li>
+          <li class="ht">
+            <img src="/Asset/html.svg" alt="HTML" /><span>HTML</span>
+          </li>
+          <li class="cs">
+            <img src="/Asset/css.svg" alt="CSS" /><span>CSS</span>
+          </li>
+        </ul>
+        <h2>Frameworks</h2>
+        <img src="/Asset/arrow.svg" class="arrow1" alt="" />
+        <h3>Skills</h3>
+        <img src="/Asset/arrow.svg" class="arrow2" alt="" />
+      </div>
```

Comment:
- [ ] Kindly note that the arrows in the `frameworks` and `skills` sections are supposed to point to the right and not down.

![image](https://user-images.githubusercontent.com/98453979/227637541-237a42d9-8c3a-4090-a73d-458e1b4a304e.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/227637663-35509fac-1fb9-40a8-a447-4a87a1f221bb.png)